content/blog/2024/09/solus-hacktoberfest-2024/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Solus Hacktoberfest 2024"
+author: "tracey"
+categories:
+- news
+date: 2024-09-30T20:29:55-05:00
+featuredimage: "/horizontal_black.png"
+url: "/2024/09/30/solus-hacktoberfest-2024"
+---
+
+Hey folks! It's spooky season, so you know what that means. It's time for [Hacktoberfest 2024](https://hacktoberfest.com/)! This is the annual event that "encourages people to contribute to open source throughout October", in exchange for rewards.
+
+This year we have two main goals. First, as always, we want to support anyone looking to get involved in Open Source contributions, or helping Solus specifically. We would love to add people to our packaging team.
```

Comment:
```suggestion
This year we have two main goals. First, as always, we want to support anyone looking to get involved in open source contributions, or helping Solus specifically. We would love to add people to our packaging team.
```

It's written as "open source" with small first letters in all other places in the post. Not sure which format is "better", but it should be uniform.

Reply:
The Microsoft style guide uses lower case, which seems more correct to me anyway. Good catch.